lib/logstash/plugin_mixins/checked_count_logger.rb

```diff
@@ -0,0 +1,35 @@
+
+module LogStash module PluginMixins
```

Comment:
IMO we should not place plugin specific (or integration specific) mixins in `LogStash::PluginMixins`, all of these could be namespaced to `LogStash::PluginMixins::Jdbc` or similar